Library/Formula/b2-tools.rb

```diff
@@ -13,12 +13,14 @@ class B2Tools < Formula
 
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec+"lib/python2.7/site-packages"
+
+    system "python", "setup.py", "install", "--prefix=#{libexec}"
```

Comment:
Oops, but when I install `b2-tools` only with below line, I got an error from `brew test b2-tools`:

```
Testing b2-tools
==> Using the sandbox
==> /usr/local/Cellar/b2-tools/0.4.4/bin/b2 authorize_account BOGUSACCTID BOGUSAPPKEY
Traceback (most recent call last):
  File "/usr/local/Cellar/b2-tools/0.4.4/libexec/bin/b2", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3130, in <module>
    @_call_aside
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3116, in _call_aside
    f(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3143, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 644, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 657, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 830, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'six>=1.10.0' distribution was not found and is required by b2
Error: b2-tools: failed
Expected /bad_auth_token/ to match "".
```

And with this line instead of the below line, `brew test b2-tools` succeeded. Then is it okay to go with this line? I'll remove the below line.